airflow/stats.py

```diff
@@ -135,19 +135,91 @@ def timing(self, stat, dt):
             return self.statsd.timing(stat, dt)
 
 
-Stats = DummyStatsLogger  # type: Any
+class SafeDogStatsdLogger:
 
-try:
-    if conf.getboolean('scheduler', 'statsd_on'):
-        from statsd import StatsClient
+    def __init__(self, dogstatsd_client, allow_list_validator=AllowListValidator()):
+        self.dogstatsd = dogstatsd_client
+        self.allow_list_validator = allow_list_validator
+
+    @validate_stat
+    def incr(self, stat, count=1, rate=1, tags=None):
+        if self.allow_list_validator.test(stat):
+            if tags is None:
+                tags = []
```

Comment:
```suggestion
                tags = tags or []
```

Reply:
I have updated the code with your suggestion.